modules/core/test/test_mat.cpp

```diff
@@ -2001,6 +2001,22 @@ TEST(Core_Vectors, issue_13078_workaround)
     ASSERT_EQ(7, ints[3]);
 }
 
+TEST(Core_MatExpr, issue_13926)
+{
+    Mat M1 = (Mat_<double>(4,4,CV_64FC1) << 1, 2, 3, 4,
+                                           5, 6, 7, 8,
+                                           9, 10, 11, 12,
+                                           13, 14, 15, 16);
+
+    Matx44d M2(1, 2, 3, 4,
+               5, 6, 7, 8,
+               9, 10, 11, 12,
+               13, 14, 15, 16);
+
+    ASSERT_EQ(0, countNonZero(M1*M2 - M1*M1));
+    ASSERT_EQ(0, countNonZero(M2*M1 - M2*M2));
```

Comment:
It is better to avoid using of `countNonZero()`, especially in case of floating-point matrixes.

Consider using this:
```
EXPECT_GE(1e-6, cvtest::norm(M1*M2, M1*M1, NORM_INF)) << Mat(M1*M2) << std::endl << Mat(M1*M1);
```

Reply:
oh sure, it's float -- my bad, i did not see it.